example/lib/solid_lints_example.dart

```diff
@@ -4,3 +4,16 @@ void main() {
 
 /// Sum of two numbers. Standard dart overflow rules apply.
 int sum(int p1, int p2) => p1 + p2;
+
+/// Check complexity fail
```

Comment:
Let's reference the metric YAML id.
```suggestion
/// Check complexity fail
/// cyclomatic_complexity_metric: max_complexity
```